package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "This is an app that helps you organize your day",
   "main": "index.js",
+  "eslintIgnore": ["index.js"],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "webpack --watch",
```

Comment:
Great work adding some configuration scripts. However, when I run `npm run build` the script seems to have a problem because am getting an error.

Kindly configure this script well. This will greatly improve your app configuration. You can do this by adding the right configuration for your build. For Example:  `"build": "webpack --mode production"` .

![scripts_requirement](https://user-images.githubusercontent.com/32361963/109607783-cf86c380-7b39-11eb-9b54-3fa3890def59.png)